TAC/Meetings/2022/2022-05-19/CCC_TAC_Minutes-2022-05-19.md

```diff
@@ -0,0 +1,161 @@
+
+### Open Action Items
+1. **[Stephen Walli, Dan Middleton, Aeva Black]** Develop a CoC escalation process, to be presented to TAC and governing board.
+    * Dan Middleton has opened a PR for review https://github.com/confidential-computing/governance/pull/92
+
+1. **[Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
+    * Open Enclave maintainers are close to 100% trained (Dave Thaler)
+    * Veracruz lead has taken the course (Thomas Fossati).
+    * Please report back that project maintainers have been made aware of the training, and have been asked to take it.
+
+1. **[Mentors]** Mentors to reach out to the project for getting the maintainers involved in Common Test Infrastructure
+
+### Completed Action Items
+2. **[Kurt Taylor]** Find out if there is a DCO needed for the Governance repo on GitHub [Issue #107] (https://github.com/confidential-computing/governance/issues/107) (Done)
+
+1. **[Kurt Taylor]** Investigate prior efforts to finalize Graduated lifecycle language, and if needed file an issue to finalize Graduated lifecycle language. (Done)
+   * Issue created to track [Issue #104] (https://github.com/confidential-computing/governance/issues/104)
+
+2. **[Kurt Taylor]** Check if terminology document markdown format will work with LF Creative team and check on their timeline for completing the formatting for the white paper (Done)
+   * Markdown is fine, timeline is 1-3 weeks, depending on length and complexity)
+
+2. **[Kurt Taylor]** Code scanning via LFX Security - add all missing projects to security for scanning, get more details on instructions on installing GitHub bot (Done) 
+    * Projects can set up the bot and repos they want scanned by following these instructions: https://community.lfx.dev/t/how-to-install-and-configure-bots-to-secure-your-projects/181
+
+### Project overviews
+### Common Test Infrastructure
+ * DT - status, noted this topic was reviewed with the Governing Board, seeking feedback from the projects, if would be used by the projects, reviewed the budget amounts
+ * DT - main cost is the LF maintainership, not test service or hardware, noted this is not a full-time job from LF
+ * DT proposed that the budget be changed and approved to 50K for LF management, 15K for test hardware
+ * DT PROPOSED: The TAC budget $50k out of the "Cons. IT Services" $100k line item, to allocate to LF management of Common Test Infrastructure, MB asked for clarification on the proposal from the project feedback, DT asked for input from mentors, no additional feedback provided for usage
+ * DM asked for specific written proposal
+
+***APPROVED*** The TAC budget $50k out of the "Cons. IT Services" $100k line item, to allocate to LF management of Common Test Infrastructure
```

Comment:
Did anyone have an AI for the next step here?

Reply:
Maybe we need to bubble out this snippet: "... DT asked for input from mentors..."
We could add a tracked action item for mentors to socialize this with their projects and find out if they have specific needs.

Reply:
As we ended up discussing next steps during the 2022-06-02 meeting we could pick up further discussion in those minutes.